libsodium-sys/src/crypto_secretbox.rs

```diff
@@ -17,6 +17,20 @@ extern {
         clen: c_ulonglong,
         n: *const [u8; crypto_secretbox_NONCEBYTES],
         k: *const [u8; crypto_secretbox_KEYBYTES]) -> c_int;
+    pub fn crypto_secretbox_detached(
+        c: *mut u8,
+        mac: *mut [u8; crypto_secretbox_xsalsa20poly1305_MACBYTES],
```

Comment:
The wrong constants are used here, `crypto_secretbox_MACBYTES` and so on should be used in the file `crypto_secretbox.rs`.